tests/in/access.wgsl

```diff
@@ -9,15 +9,19 @@ struct Bar {
 [[group(0), binding(0)]]
 var<storage> bar: [[access(read_write)]] Bar;
 
+[[group(0), binding(1)]]
+var<storage> qux: Bar;
```

Comment:
any reason to have both the new and the old syntax (for `bar`) in the same test? If we are updating the syntax, let's not leave any trace behind